Failures in [node-test-pull-request/69716](https://ci.nodejs.org/job/node-test-pull-request/69716/) to [node-test-pull-request/69815](https://ci.nodejs.org/job/node-test-pull-request/69815/) that failed 2 or more PRs
(Generated with `ncu-ci walk pr --stats=true --markdown reports/2025-10-19.md`)

| UTC Time         | RUNNING | SUCCESS | UNSTABLE | ABORTED | FAILURE | Green Rate |
| ---------------- | ------- | ------- | -------- | ------- | ------- | ---------- |
| 2025-10-19 00:12 | 0       | 10      | 17       | 1       | 72      | 10.10%     |


### Jenkins Failure

| Reason | <code>Build timed out (after 30 minutes). Marking the build as failed.</code> |
| - | :- |
| Type | JENKINS_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/60251/](https://ci.nodejs.org/job/node-test-pull-request/69719/), [https://github.com/nodejs/node/pull/60255/](https://ci.nodejs.org/job/node-test-pull-request/69805/)) |
| Appeared | [test-azure_msft-win11_vs2022-arm64-5](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/37323/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69719/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69805/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/37323/console">Example</a></summary>

```
Build timed out (after 30 minutes). Marking the build as failed.
```
</details>

-------


### JSTest Failure

| Reason | <code>parallel/test-without-async-context-frame</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 17 ([https://github.com/nodejs/node/pull/59929/](https://ci.nodejs.org/job/node-test-pull-request/69716/), [https://github.com/nodejs/node/pull/60251/](https://ci.nodejs.org/job/node-test-pull-request/69719/), [https://github.com/nodejs/node/pull/59896/](https://ci.nodejs.org/job/node-test-pull-request/69721/), [https://github.com/nodejs/node/pull/60255/](https://ci.nodejs.org/job/node-test-pull-request/69733/), [https://github.com/nodejs/node/pull/60250/](https://ci.nodejs.org/job/node-test-pull-request/69746/), [https://github.com/nodejs/node/pull/60254/](https://ci.nodejs.org/job/node-test-pull-request/69763/), [https://github.com/nodejs/node/pull/60256/](https://ci.nodejs.org/job/node-test-pull-request/69764/), [https://github.com/nodejs/node/pull/60244/](https://ci.nodejs.org/job/node-test-pull-request/69765/), [https://github.com/nodejs/node/pull/60217/](https://ci.nodejs.org/job/node-test-pull-request/69766/), [https://github.com/nodejs/node/pull/60278/](https://ci.nodejs.org/job/node-test-pull-request/69773/), [https://github.com/nodejs/node/pull/60205/](https://ci.nodejs.org/job/node-test-pull-request/69778/), [https://github.com/nodejs/node/pull/60266/](https://ci.nodejs.org/job/node-test-pull-request/69780/), [https://github.com/nodejs/node/pull/60271/](https://ci.nodejs.org/job/node-test-pull-request/69781/), [https://github.com/nodejs/node/pull/60277/](https://ci.nodejs.org/job/node-test-pull-request/69785/), [https://github.com/nodejs/node/pull/60272/](https://ci.nodejs.org/job/node-test-pull-request/69786/), [https://github.com/nodejs/node/pull/60269/](https://ci.nodejs.org/job/node-test-pull-request/69793/), [https://github.com/nodejs/node/pull/60289/](https://ci.nodejs.org/job/node-test-pull-request/69799/)) |
| Appeared | [vm-xrtit](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67444/console), [vm-qeafq](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67435/console), [vm-4tv35](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67433/console), [vm-oa16m](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67427/console), [vm-bmoqy](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67419/console), [test-digitalocean-ubuntu2404_sharedlibs_container-x64-6](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2404_sharedlibs_shared_x64/53203/console), [test-digitalocean-ubuntu2404_sharedlibs_container-x64-8](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2404_sharedlibs_shared_x64/53197/console), [vm-gy321](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67409/console), [vm-imm7g](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67408/console), [test-digitalocean-ubuntu2404_sharedlibs_container-x64-9](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2404_sharedlibs_shared_x64/53195/console), [vm-gnhs2](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67406/console), [test-digitalocean-ubuntu2404_sharedlibs_container-x64-2](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2404_sharedlibs_shared_x64/53193/console), [vm-oiqpl](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67401/console), [vm-263lv](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67399/console), [vm-iv9zi](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67388/console), [vm-tn28e](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67387/console), [vm-o3mm0](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67376/console), [vm-qfxrp](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67373/console), [vm-c9pt9](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67372/console), [vm-4ckp1](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67359/console), [vm-36l2p](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67358/console), [vm-gtwyt](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67351/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69716/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69799/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67444/console">Example</a></summary>

```
not ok 4167 parallel/test-without-async-context-frame
  ---
  duration_ms: 120040.46100
  severity: fail
  exitcode: -15
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>sequential/test-worker-prof</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 13 ([https://github.com/nodejs/node/pull/60251/](https://ci.nodejs.org/job/node-test-pull-request/69731/), [https://github.com/nodejs/node/pull/60256/](https://ci.nodejs.org/job/node-test-pull-request/69732/), [https://github.com/nodejs/node/pull/60255/](https://ci.nodejs.org/job/node-test-pull-request/69733/), [https://github.com/nodejs/node/pull/60205/](https://ci.nodejs.org/job/node-test-pull-request/69735/), [https://github.com/nodejs/node/pull/60250/](https://ci.nodejs.org/job/node-test-pull-request/69750/), [https://github.com/nodejs/node/pull/59873/](https://ci.nodejs.org/job/node-test-pull-request/69752/), [https://github.com/nodejs/node/pull/60276/](https://ci.nodejs.org/job/node-test-pull-request/69759/), [https://github.com/nodejs/node/pull/60244/](https://ci.nodejs.org/job/node-test-pull-request/69765/), [https://github.com/nodejs/node/pull/60217/](https://ci.nodejs.org/job/node-test-pull-request/69766/), [https://github.com/nodejs/node/pull/60254/](https://ci.nodejs.org/job/node-test-pull-request/69770/), [https://github.com/nodejs/node/pull/60269/](https://ci.nodejs.org/job/node-test-pull-request/69779/), [https://github.com/nodejs/node/pull/60266/](https://ci.nodejs.org/job/node-test-pull-request/69796/), [https://github.com/nodejs/node/pull/60289/](https://ci.nodejs.org/job/node-test-pull-request/69814/)) |
| Appeared | [test-azure_msft-win10_vs2019-x64-3](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win10-COMPILED_BY-vs2022_clang/37332/console), [test-azure_msft-win10_vs2019-x64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win10-COMPILED_BY-vs2022_clang/37326/console), [test-azure_msft-win11_vs2022-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win11-COMPILED_BY-vs2022_clang/37322/console), [test-azure_msft-win10_vs2019-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win10-COMPILED_BY-vs2022_clang/37304/console), [test-azure_msft-win11_vs2022-x64-3](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win11-COMPILED_BY-vs2022_clang/37304/console), [test-azure_msft-win11_vs2022-arm64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/37291/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69731/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69814/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win10-COMPILED_BY-vs2022_clang/37332/console">Example</a></summary>

```
not ok 1145 sequential/test-worker-prof
  ---
  duration_ms: 2510.79000
  severity: fail
  exitcode: 1
  stack: |-
    files found in working directory before worker starts [ 'isolate-000001E57D63B000-1212-v8.log' ]
    parent prof file: isolate-000001E57D63B000-1212-v8.log
    parent posted 36506 messages
    files found in working directory before worker exits [
      'isolate-000001E57D63B000-1212-v8.log',
      'isolate-000001E57EC22000-1212-v8.log'
    ]
    worker prof file: isolate-000001E57EC22000-1212-v8.log
    
    worker ticks 22
    parent ticks 14
    node:internal/assert/utils:146
      throw error;
      ^
    
    AssertionError [ERR_ASSERTION]: parent ticks <= 15:
    tick,0x7ff75d3df6e5,163800,0,0x0,0,0x7ff75d2eb0c0,0x1a490c04917,0x2539c386dac,0x2539c3860ca,0x2539c385e3b,0x2539c38684c,0xebdd4c1330,0x2539c3830b3,0x2539c3877fa,0xebdd4f06b8,0x17f071492a6,0x2539c388277,0x2539c388b52,0x2539c387687,0x2539c3869c9,0xebdd4c1330,0x2539c3830b3,0x2539c39155f,0x17f07147490
    tick,0x7ff75d42f394,243899...
```
</details>

-------

| Reason | <code>parallel/test-runner-output</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 10 ([https://github.com/nodejs/node/pull/60251/](https://ci.nodejs.org/job/node-test-pull-request/69719/), [https://github.com/nodejs/node/pull/60217/](https://ci.nodejs.org/job/node-test-pull-request/69720/), [https://github.com/nodejs/node/pull/60236/](https://ci.nodejs.org/job/node-test-pull-request/69723/), [https://github.com/nodejs/node/pull/60255/](https://ci.nodejs.org/job/node-test-pull-request/69724/), [https://github.com/nodejs/node/pull/59896/](https://ci.nodejs.org/job/node-test-pull-request/69728/), [https://github.com/nodejs/node/pull/60245/](https://ci.nodejs.org/job/node-test-pull-request/69741/), [https://github.com/nodejs/node/pull/60205/](https://ci.nodejs.org/job/node-test-pull-request/69762/), [https://github.com/nodejs/node/pull/60271/](https://ci.nodejs.org/job/node-test-pull-request/69775/), [https://github.com/nodejs/node/pull/60131/](https://ci.nodejs.org/job/node-test-pull-request/69792/), [https://github.com/nodejs/node/pull/60266/](https://ci.nodejs.org/job/node-test-pull-request/69796/)) |
| Appeared | [test-azure_msft-win11_vs2022-x64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-COMPILED_BY-vs2022_clang/37319/console), [test-osuosl-aix72-ppc64_be-4](https://ci.nodejs.org/job/node-test-commit-aix/nodes=aix72-ppc64/59443/console), [vm-oa16m](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67421/console), [vm-gnhs2](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67406/console), [undefined](https://ci.nodejs.org/job/node-test-commit-custom-suites-freestyle/44412/console), [vm-wi81c](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx13-x64/67356/console), [test-ibm-rhel9-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=rhel9-x64/66985/console), [vm-utd49](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67361/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69719/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69796/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-COMPILED_BY-vs2022_clang/37319/console">Example</a></summary>

```
not ok 787 parallel/test-runner-output
  ---
  duration_ms: 4465.99500
  severity: fail
  exitcode: 1
  stack: |-
    Test failure: 'test-runner/output/test-timeout-flag.js'
    Location: test\parallel\test-runner-output.mjs:356:5
    AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:
    + actual - expected
    ... Skipped lines
    
      'TAP version 13\n' +
        '# Subtest: --test-timeout is set to 100ms\n' +
        '    # Subtest: should timeout after 100ms\n' +
        '    not ok 1 - should timeout after 100ms\n' +
        '      ---\n' +
    ...

```
</details>

-------

| Reason | <code>parallel/test-quic-test-client</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 7 ([https://github.com/nodejs/node/pull/60245/](https://ci.nodejs.org/job/node-test-pull-request/69748/), [https://github.com/nodejs/node/pull/60250/](https://ci.nodejs.org/job/node-test-pull-request/69750/), [https://github.com/nodejs/node/pull/60272/](https://ci.nodejs.org/job/node-test-pull-request/69755/), [https://github.com/nodejs/node/pull/60266/](https://ci.nodejs.org/job/node-test-pull-request/69757/), [https://github.com/nodejs/node/pull/60217/](https://ci.nodejs.org/job/node-test-pull-request/69766/), [https://github.com/nodejs/node/pull/60277/](https://ci.nodejs.org/job/node-test-pull-request/69785/), [https://github.com/nodejs/node/pull/60178/](https://ci.nodejs.org/job/node-test-pull-request/69808/)) |
| Appeared | [vm-92kcy](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67452/console), [vm-qeafq](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67432/console), [test-mnx-smartos23-x64-6](https://ci.nodejs.org/job/node-test-commit-smartos/nodes=smartos23-x64/62847/console), [vm-oiqpl](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67401/console), [vm-263lv](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67399/console), [vm-jovkh](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67391/console), [vm-j9sr5](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67389/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69748/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69808/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67452/console">Example</a></summary>

```
not ok 2573 parallel/test-quic-test-client
  ---
  duration_ms: 1435.47400
  severity: fail
  exitcode: 1
  stack: |-
    /Users/admin/build/workspace/node-test-commit-osx/nodes/macos15-x64/out/Release/ngtcp2_test_client
    node:internal/modules/run_main:107
        triggerUncaughtException(
        ^
    
    AssertionError [ERR_ASSERTION]: Missing expected rejection.
        at process.processTicksAndRejections (node:internal/process/task_queues:103:5)
        at async file:///Users/admin/build/workspace/node-test-commit-osx/nodes/macos15-x64/test/parallel/test-quic-test-client.mjs:36:1 {
      generatedMessage: false,
      code: 'ERR_ASSERTION',
      actual: undefined,
      expected: { message: /Process exited with code 1 and signal null/ },
      operator: 'rejects',
      diff: 'simple'
    }
    
    Node.js v26.0.0-pre
  ...

```
</details>

-------

| Reason | <code>parallel/test-runner-watch-mode-kill-signal</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 6 ([https://github.com/nodejs/node/pull/60217/](https://ci.nodejs.org/job/node-test-pull-request/69720/), [https://github.com/nodejs/node/pull/60256/](https://ci.nodejs.org/job/node-test-pull-request/69736/), [https://github.com/nodejs/node/pull/60244/](https://ci.nodejs.org/job/node-test-pull-request/69765/), [https://github.com/nodejs/node/pull/60178/](https://ci.nodejs.org/job/node-test-pull-request/69777/), [https://github.com/nodejs/node/pull/60271/](https://ci.nodejs.org/job/node-test-pull-request/69789/), [https://github.com/nodejs/node/pull/60205/](https://ci.nodejs.org/job/node-test-pull-request/69790/)) |
| Appeared | [vm-4tv35](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67437/console), [vm-oa16m](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67436/console), [vm-bmoqy](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67424/console), [vm-qeafq](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67418/console), [vm-gy321](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67409/console), [vm-gnhs2](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67406/console), [vm-se52j](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67377/console), [test-digitalocean-alpine321_container-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=alpine-last-latest-x64/66984/console), [test-azure-ubuntu2204_container-arm64-6](https://ci.nodejs.org/job/node-test-commit-arm-debug/nodes=ubuntu2204_debug-arm64/20696/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69720/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69790/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67437/console">Example</a></summary>

```
not ok 3491 parallel/test-runner-watch-mode-kill-signal
  ---
  duration_ms: 120102.45300
  severity: fail
  exitcode: -15
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>sequential/test-perf-hooks</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 5 ([https://github.com/nodejs/node/pull/60256/](https://ci.nodejs.org/job/node-test-pull-request/69736/), [https://github.com/nodejs/node/pull/60245/](https://ci.nodejs.org/job/node-test-pull-request/69744/), [https://github.com/nodejs/node/pull/60205/](https://ci.nodejs.org/job/node-test-pull-request/69762/), [https://github.com/nodejs/node/pull/60269/](https://ci.nodejs.org/job/node-test-pull-request/69788/), [https://github.com/nodejs/node/pull/60289/](https://ci.nodejs.org/job/node-test-pull-request/69799/)) |
| Appeared | [vm-xrtit](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67444/console), [vm-qeafq](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67435/console), [vm-gnhs2](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67406/console), [vm-l4ydv](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67400/console), [vm-iv9zi](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67385/console), [vm-se52j](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67377/console), [vm-o3mm0](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67376/console), [test-digitalocean-ubuntu2404_sharedlibs_container-x64-10](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2404_sharedlibs_openssl30_x64/53163/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69736/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69799/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67444/console">Example</a></summary>

```
not ok 4765 sequential/test-perf-hooks
  ---
  duration_ms: 817.95400
  severity: fail
  exitcode: 1
  stack: |-
    node:internal/assert/utils:146
      throw error;
      ^
    
    AssertionError [ERR_ASSERTION]: Date.now() - performance.timeOrigin (417.280029296875) - process.uptime() * 1000 (289.587266) = 127.692763296875 >= +- 50
        at Object.<anonymous> (/Users/admin/build/workspace/node-test-commit-osx/nodes/macos15-x64/test/sequential/test-perf-hooks.js:36:3)
        at Module._compile (node:internal/modules/cjs/loader:1760:14)
        at Object..js (node:internal/modules/cjs/loader:1892:10)
        at Module.load (node:internal/modules/cjs/loader:1480:32)
        at Module._load (node:internal/modules/cjs/loader:1299:12)
        at TracingChannel.traceSync (node:diagnostics_channel:328:14)
        at wrapModuleLoad (node:internal/modules/cjs/loader:245:24)
        at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:154:5)
        at node:internal/main/run_main_module:33...
```
</details>

-------

| Reason | <code>wpt/test-hr-time</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 5 ([https://github.com/nodejs/node/pull/59929/](https://ci.nodejs.org/job/node-test-pull-request/69716/), [https://github.com/nodejs/node/pull/59896/](https://ci.nodejs.org/job/node-test-pull-request/69728/), [https://github.com/nodejs/node/pull/60261/](https://ci.nodejs.org/job/node-test-pull-request/69734/), [https://github.com/nodejs/node/pull/60266/](https://ci.nodejs.org/job/node-test-pull-request/69787/), [https://github.com/nodejs/node/pull/60271/](https://ci.nodejs.org/job/node-test-pull-request/69803/)) |
| Appeared | [vm-5wwhg](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67447/console), [test-azure_msft-win11_vs2022-arm64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/37316/console), [vm-oiqpl](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67401/console), [vm-ztsar](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67374/console), [vm-4jmql](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67366/console), [vm-gtwyt](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67351/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69716/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69803/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67447/console">Example</a></summary>

```
not ok 4806 wpt/test-hr-time
  ---
  duration_ms: 2335.93000
  severity: fail
  exitcode: 1
  stack: |-
    [SKIPPED] idlharness-shadowrealm.window.js: ShadowRealm support is not enabled
    [SKIPPED] window-worker-timeOrigin.window.js: depends on URL.createObjectURL(blob)
    [PASS] self.performance.now() is a function that returns a number
    [PASS] self.performance.now() returns a positive number
    [PASS] self.performance.now() difference is not negative
    [PASS] Performance interface extends EventTarget.
    [PASS] self.performance.now() returns a positive number
    [PASS] self.performance.now() difference is not negative
    [PASS] idl_test validation
    [PASS] Partial interface mixin WindowOrWorkerGlobalScope: original interface mixin defined
    [PASS] Partial interface mixin WindowOrWorkerGlobalScope: member names are unique
    [PASS] Partial interface Window: member names are unique
    [PASS] Window includes GlobalEventHandlers: member names are unique
    [PASS] Window includes WindowEventH...
```
</details>

-------

| Reason | <code>client-proxy/test-https-proxy-request-invalid-char-in-url</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 4 ([https://github.com/nodejs/node/pull/59896/](https://ci.nodejs.org/job/node-test-pull-request/69721/), [https://github.com/nodejs/node/pull/60250/](https://ci.nodejs.org/job/node-test-pull-request/69738/), [https://github.com/nodejs/node/pull/60178/](https://ci.nodejs.org/job/node-test-pull-request/69777/), [https://github.com/nodejs/node/pull/60266/](https://ci.nodejs.org/job/node-test-pull-request/69780/)) |
| Appeared | [vm-oa16m](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67427/console), [vm-bmoqy](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67424/console), [vm-cmeeo](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67379/console), [vm-4ckp1](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67359/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69721/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69780/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67427/console">Example</a></summary>

```
not ok 4378 client-proxy/test-https-proxy-request-invalid-char-in-url
  ---
  duration_ms: 533.88300
  severity: fail
  exitcode: 1
  stack: |-
    [Upstream server] responding to request for '/newline-in-host'
    [Upstream server] responding to request for '/carriage-return-in-host'
    [Upstream server] responding to request for '/carriage-return-in-port'
    [Upstream server] responding to request for '/crlf-in-host'
    [Proxy client] Received response from server for 'https://local\nhost:61468/newline-in-host': Response for '/newline-in-host'
    [Proxy client] #1 closed request for: 'https://local\nhost:61468/newline-in-host'
    [Proxy client] Received response from server for 'https://local\rhost:61468/carriage-return-in-host': Response for '/carriage-return-in-host'
    [Proxy client] #2 closed request for: 'https://local\rhost:61468/carriage-return-in-host'
    [Upstream server] responding to request for '/newline-in-port'
    [Proxy client] Received response from server for 'https://localhost:6\r1...
```
</details>

-------

| Reason | <code>parallel/test-http2-close-while-writing</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 4 ([https://github.com/nodejs/node/pull/60217/](https://ci.nodejs.org/job/node-test-pull-request/69720/), [https://github.com/nodejs/node/pull/60269/](https://ci.nodejs.org/job/node-test-pull-request/69779/), [https://github.com/nodejs/node/pull/60271/](https://ci.nodejs.org/job/node-test-pull-request/69794/), [https://github.com/nodejs/node/pull/60289/](https://ci.nodejs.org/job/node-test-pull-request/69799/)) |
| Appeared | [vm-xrtit](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67444/console), [vm-ehfzr](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67441/console), [vm-qeafq](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67426/console), [vm-36l2p](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67358/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69720/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69799/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67444/console">Example</a></summary>

```
not ok 2139 parallel/test-http2-close-while-writing
  ---
  duration_ms: 120120.03000
  severity: fail
  exitcode: -15
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>parallel/test-runner-mock-timers-scheduler</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/60256/](https://ci.nodejs.org/job/node-test-pull-request/69732/), [https://github.com/nodejs/node/pull/60205/](https://ci.nodejs.org/job/node-test-pull-request/69778/)) |
| Appeared | [vm-oa16m](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67425/console), [vm-c9pt9](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67372/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69732/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69778/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67425/console">Example</a></summary>

```
not ok 2750 parallel/test-runner-mock-timers-scheduler
  ---
  duration_ms: 889.94500
  severity: fail
  exitcode: 1
  stack: |-
    Test failure: 'should advance in time and trigger timers when calling the .tick function'
    Location: test/parallel/test-runner-mock-timers-scheduler.js:10:3
    AssertionError [ERR_ASSERTION]: time should be advanced less than the 100ms
        at /Users/admin/build/workspace/node-test-commit-osx/nodes/macos15-x64/test/parallel/test-runner-mock-timers-scheduler.js:21:14
        at /Users/admin/build/workspace/node-test-commit-osx/nodes/macos15-x64/test/common/index.js:475:15
        at async Test.run (node:internal/test_runner/test:1113:7)
        at async Promise.all (index 0)
        at async Suite.run (node:internal/test_runner/test:1518:7)
        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:358:3) {
      generatedMessage: false,
      code: 'ERR_ASSERTION',
      actual: false,
      expected: true,
      operator: '==',
      diff: 'simple'
  ...
```
</details>

-------


### Build Failure

| Reason | <code>fatal: No rebase in progress?</code> |
| - | :- |
| Type | BUILD_FAILURE |
| Failed PR | 5 ([https://github.com/nodejs/node/pull/60271/](https://ci.nodejs.org/job/node-test-pull-request/69754/), [https://github.com/nodejs/node/pull/60230/](https://ci.nodejs.org/job/node-test-pull-request/69791/), [https://github.com/nodejs/node/pull/60255/](https://ci.nodejs.org/job/node-test-pull-request/69805/), [https://github.com/nodejs/node/pull/60289/](https://ci.nodejs.org/job/node-test-pull-request/69809/), [https://github.com/nodejs/node/pull/60298/](https://ci.nodejs.org/job/node-test-pull-request/69810/)) |
| Appeared | [test-mnx-ubuntu2204-x64-1](https://ci.nodejs.org/job/node-test-linter/62233/console), [test-ibm-ubuntu2404_sharedlibs_container-x64-5](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2404_sharedlibs_shared_x64/53211/console), [test-ibm-ubuntu2404_sharedlibs_container-x64-4](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2404_sharedlibs_shared_x64/53186/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69754/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69810/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-linter/62233/console">Example</a></summary>

```
fatal: No rebase in progress?
```
</details>

-------

